blockchain/vm/errors.go

```diff
@@ -36,6 +36,8 @@ var (
 	ErrTotalTimeLimitReached             = errors.New("reached the total execution time limit for txs in a block")
 	ErrOpcodeComputationCostLimitReached = errors.New("reached the opcode computation cost limit")
 	ErrFailedOnSetCode                   = errors.New("failed on setting code to an account")
+	ErrOutOfGas                          = errors.New("out of gas")
+	ErrGasUintOverflow                   = errors.New("gas uint64 overflow")
```

Comment:
What about adding it to kerrors?

Reply:
There seems to be an existing function, so we'll use that.
https://github.com/kaiachain/kaia/pull/119#discussion_r1830259459